business/workloads.go

```diff
@@ -620,115 +620,146 @@ func fetchWorkloads(ctx context.Context, layer *Layer, namespace string, labelSe
 	// Pods are always fetched
 	go func() {
 		defer wg.Done()
-		var err error
-		pods, err = layer.Workload.k8s.GetPods(namespace, labelSelector)
-		if err != nil {
-			log.Errorf("Error fetching Pods per namespace %s: %s", namespace, err)
-			errChan <- err
+		for _, c := range clusterNames {
```

Comment:
I was thinking, the workload list should be fetched with the user credentials and not with the kiali SA token? 
Not sure if we should instantiate a layer for each cluster and get the data from there? As I see the layer has its own k8s client.